.gitignore

```diff
@@ -5,4 +5,5 @@ build/*
 !build/.gitkeep
 .php_cs.cache
 infection-cache
-.travis/infection-private.pem
\ No newline at end of file
+.travis/infection-private.pem
+infection.json
```

Comment:
The order of entries in `.gitignore` appears to be a bit random; please let me know if there is a specific rule you would like me to follow, and then I will move this entry into the appropriate place.